facia/app/views/fragments/emailFrontBody.scala.html

```diff
@@ -71,19 +71,29 @@ <h4 class="trail-text">@Html(trailText)</h4>
     }
 }
 
-@faciaCardSmall(card: ContentCard) = {
+@faciaCardSmall(card: ContentCard, withImage: Boolean) = {
     @paddedRow(Seq(toneClassFromStyle(card.cardStyle))) {
-        @fullRow(Seq("facia-card", "facia-card__text", "facia-card__text--last")) {
-            @headline(card)
-        }
+        <div class="facia-card">
+            @if(withImage) {
+                @fullRowWithSubColumns() {
+                    @subColumn(7, Seq("facia-card__text")) { @headline(card)}
+                    @subColumn(5, Seq("facia-card__text"), last = true) { @imgFromCard(card, 5) }
```

Comment:
What's supposed to happen if there's no image for some reason?

![image](https://cloud.githubusercontent.com/assets/3072877/22259869/e9d01110-e25e-11e6-9178-372641cac12f.png)